### PR TITLE
Fix using new registration-operator fail e2e test issue

### PR DESCRIPTION
**Pull request by xuezhaojun**
Signed-off-by: xuezhaojun <zxue@redhat.com>

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -65,7 +65,8 @@ export KUBECONFIG=</path/to/kubeconfig>
 
 make images
 
-kind load docker-image quay.io/open-cluster-management-io/helloworld-addon:latest --name {your cluster name}
+kind load docker-image quay.io/open-cluster-management/addon-framework --name {your cluster name}
```
Comment (qiujian16) [inline CONTRIBUTING.md]: I do not think we have this image.

File: Makefile [hunk 2]
```diff
@@ -45,11 +45,8 @@ $(call add-bindata,foundation-agent,./examples/helloworld/manifests/...,bindata,
 # It will generate target "image-$(1)" for building the image and binding it as a prerequisite to target "images".
 $(call build-image,$(IMAGE),$(IMAGE_REGISTRY)/$(IMAGE),./Dockerfile,.)
 
-deploy-hub:
-	examples/deploy/managedcluster/hub/install.sh
-
-deploy-klusterlet:
-	examples/deploy/managedcluster/klusterlet/install.sh
+deploy-all-in-one-env:
```
Comment (qiujian16) [inline Makefile]: deploy-ocm:
   examples/deploy/ocm/install.sh

File: examples/deploy/all-in-one/install.sh [hunk 3]
```diff
@@ -50,13 +50,28 @@ for i in {1..7}; do
...
 
-echo "############  ManagedCluster hub is installed successfully!!"
+echo "############  All-in-one env is installed successfully!!"
 
 echo "############  Cleanup"
 cd ../ || exist
 rm -rf registration-operator
 
-echo "############  Finished installation!!!"
+echo "############  Finished installation!!!"
```
Comment (qiujian16) [inline examples/deploy/all-in-one/install.sh]: eof

**Review by qiujian16 — COMMENTED** [review]
thanks, this looks pretty neat.

File: CONTRIBUTING.md (hunk 1 above)
Comment (xuezhaojun) [reply CONTRIBUTING.md]: Got it.

**Pull request merged**